app/frontend/src/services/tasks.ts

```diff
@@ -56,3 +58,14 @@ export async function updateTask(data: UpdateTaskMessagePayload): Promise<Update
 
   return response as UpdateTaskResponsePayload;
 }
+
+/**
+ * Updates task status
+ *
+ * @param data - update params
+ */
+export async function changeTaskStatus(data: ChangeTaskStatusPayload): Promise<ChangeTaskStatusResponsePayload> {
```

Comment:
This method is responsible for the communication with API. So the `ChangeTaskStatusPayload` is the **only** agreement between client and API. 

This method should **compose** the `ChangeTaskStatusPayload` because only it should know about how to pass data to the API. No one outside this method should worry about that.